cratedb_toolkit/cluster/core.py

```
@@ -599,6 +603,30 @@ def load_table(
+        Export data from a database table on a standalone CrateDB Server.
+
+        Synopsis
+        --------
+        export CRATEDB_CLUSTER_URL=crate://crate@localhost:4200/testdrive/demo
+
+        ctk load table influxdb2://example:token@localhost:8086/testdrive/demo
+        ctk load table mongodb://localhost:27017/testdrive/demo
+        """
+        source_url = self.address.dburi
+        target_url_obj = URL(target.url)
+        #source_url = source.url
```

Ruff (0.11.9)

Found commented-out code

Remove commented-out code

(ERA001)